### PR TITLE
Mag 23

### PR DESCRIPTION
**Pull request by cstech**
Made country submit to the API without a region

### CODE REVIEW

File: Model/DutyCalculator.php [hunk 1]
```diff
@@ -132,7 +132,7 @@ public function getDutyandTax($cartId, $shippingCharge, $shippingMethodCode, $sh
                 return $this->response;
             }
            
-            if (empty($address->getCountryId()) || empty($address->getRegionCode())) {
```
Comment (lizzyWithReach) [inline Model/DutyCalculator.php:135]: I don't understand how this works, if the country is empty then we got the duties and taxes? And this is an OR statement, so it should have worked before right?

**Review by lizzyWithReach — CHANGES_REQUESTED** [review]
I'm just not sure why this code change works, if you could provide more context that would be great. I'm guessing I'm missing something

**Comment by cstech** [comment]
Going to remove block of code

File: Model/DutyCalculator.php (hunk 1 above)
Comment (lizzyWithReach) [reply Model/DutyCalculator.php:135]: Clarified with Ervin, if the country is empty (which it won't ever be) then the duties would be reset to 0 and we would never call DHL. That's what this block is doing

**Pull request closed without merging**